sys/include/iec62056/obis.h

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (C) 2023 Koen Zandberg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    sys_iec62056_obis OBIS
+ * @ingroup     sys_iec62056
+ * @brief       OBject Identification System Cmpanion Specification for Energy Monitoring
+ *
+ * This module mainly contains functions related to parsing OBIS codes from iec62056-21 modes.
+ *
+ * @see https://en.wikipedia.org/wiki/IEC_62056
+ *
+ * @{
+ *
+ * @file
+ *
+ * @author      Koen Zandberg <koen@bergzand.net>
+ */
+
+#ifndef IEC62056_OBIS_H
+#define IEC62056_OBIS_H
+
+#include <stdint.h>
+#include <stddef.h>
+#include <stdbool.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+/**
+ * @brief Value group A definitions
+ */
+typedef enum {
+    COSEM_OBIS_GROUP_A_ABSTRACT     = 0,    /**< Abstract objects */
+    COSEM_OBIS_GROUP_A_ELECTRICITY  = 1,    /**< Electricity related */
+    COSEM_OBIS_GROUP_A_HEAT_COST    = 4,    /**< Heat cost allocater related */
+    COSEM_OBIS_GROUP_A_THERMAL_1    = 5,    /**< Thermal energy related */
+    COSEM_OBIS_GROUP_A_THERMAL_2    = 6,    /**< More thermal energy related */
+    COSEM_OBIS_GROUP_A_GAS          = 7,    /**< Gas related */
+    COSEM_OBIS_GROUP_A_COLD_WATER   = 8,    /**< Cold water related */
+    COSEM_OBIS_GROUP_A_HOT_WATER    = 9,    /**< Hot water related */
+    COSEM_OBIS_GROUP_A_OTHER        = 15,   /**< Other media */
+} iec62056_obis_group_a_t;
+
+/**
+ * @brief OBIS identifier struct
+ */
+typedef struct {
+    uint8_t a;  /**< Group A: media/energy type */
```

Comment:
```suggestion
    uint8_t a;  /**< Group A: Media/energy type */
```

Or, remove capital from the other group names.